packages/blocks/src/__internal__/clipboard/utils.ts

```diff
@@ -15,6 +15,7 @@ import markdownUtils from './markdown-utils.js';
 export enum CLIPBOARD_MIMETYPE {
   HTML = 'text/html',
   TEXT = 'text/plain',
+  BLOCKSUITE_SURFACE = 'blocksuite/surface',
```

Comment:
There may be system compatibility issues when using custom mimetypes. For more details, you can consult with @QiShaoXuan 

Related to https://github.com/toeverything/blocksuite/pull/1936

Reply:
Yeah. It's the playwright's bug.When using the headed mode, tests pass. Link issue: https://github.com/microsoft/playwright/issues/18013
And there is a draft code for current testing. I will fix it today.